src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    static Task[] taskList = new Task[100];
+    static Boolean[] taskStatusList = new Boolean [100];
+    static int currentCount = 0;
+    static String dashedLine = "\t____________________________________________________________";
+
+    public static void markTask(int index){
```

Comment:
You could add spaces between the methods and the braces to be more consistent with the K&R style